mmap-alloc/src/lib.rs

```diff
@@ -107,14 +159,33 @@ impl MapAllocBuilder {
         } else {
             self.pagesize
         };
+        let obj_align = if let Some(obj_align) = self.obj_align {
+            assert!(
+                obj_align >= self.pagesize,
+                "object alignment ({}) is smaller than the page size ({})",
```

Comment:
What is the purpose of this assert? So long as obj_align is a power of 2, we automatically support it if it is smaller than the page size.

Reply:
Yeah I suppose that's fair. I'll remove it.